helmfile/charts/rclone-sync/files/rclone.conf

```diff
@@ -0,0 +1,25 @@
+[{{ .Values.config.source.name }}]
+type = {{ .Values.config.source.type }}
+env_auth = false
+{{- if eq .Values.config.source.type "s3" }}
+access_key_id = {{ .Values.config.source.s3.accessKey }}
+secret_access_key = {{ .Values.config.source.s3.secretKey }}
+region = {{ .Values.config.source.s3.region }}
+endpoint = {{ .Values.config.source.s3.regionEndpoint }}
+force_path_style = {{ .Values.config.source.s3.forcePathStyle }}
+v2_auth = {{ .Values.config.source.s3.v2Auth }}
+acl = private
```

Comment:
Maybe we can just not set this option and let it be [default](https://rclone.org/s3/#s3-acl)?